packages/gafl-webapp-service/src/pages/summary/contact-summary/route.js

```diff
@@ -112,9 +112,29 @@ export class RowGenerator {
 
   generateAddressRow (countryName) {
     const { licensee } = this.permission
-    const text = [licensee.premises, licensee.street, licensee.locality, licensee.town, licensee.postcode, countryName?.toUpperCase()]
-      .filter(Boolean)
-      .join(', ')
+
+    const titleCase = str => {
+      if (typeof str !== 'string') {
+        return ''
+      }
+
+      return str
+        .toLowerCase()
+        .split(' ')
+        .map(word => word.charAt(0).toUpperCase() + word.slice(1))
+        .join(' ')
+    }
+
+    const formattedParts = [
+      titleCase(licensee.premises),
+      titleCase(licensee.street),
+      titleCase(licensee.locality),
+      titleCase(licensee.town),
+      licensee.postcode.toUpperCase(),
+      countryName.toUpperCase()
+    ]
+
+    const text = formattedParts.filter(Boolean).join(', ')
```

Comment:
Honestly, this seems overly complex. What I think you should do is `toLowerCase()` on premises, street, locality and town, then in the browser set a style on that element of `text-transform: uppercase`. That's what we do in address lookup service when we format it for display when the user is selecting an address: https://github.com/DEFRA/rod-licensing/blob/435e74ebb124eeda4a409f5aa256ba188125f06b/packages/gafl-webapp-service/src/services/address-lookup/address-lookup-service.js#L41